src/cache-redis/cache-redis.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from "@nestjs/common";
+import axios from "axios";
+
+@Injectable()
+export class CacheRedisService {
+	constructor() {}
+	  async	getExchange() {
+      const res = await axios.get(`${process.env.REDIS_STORE_URL}/cache`, {
```

Comment:
Use the `ProcessEnvModule` and the `ProcessEnvProxy` class. Accessing `process.env` directly removes the capability to unit test in the future.

Reply:
Okay Noted!
will enhance with debio-1180[https://blocksphere2020.atlassian.net/browse/DBIO-1180](https://blocksphere2020.atlassian.net/browse/DBIO-1180)